### PR TITLE
feat: create parity issue workflow

### PR DESCRIPTION
**Pull request by joshgummersall**
This workflow will create a parity issue that includes a short summary
and a link to a pull request, along with the parity label, to keep us
honest!

It will trigger on manual dispatch - actions will need to be added to
other repos to trigger this workflow.

### CODE REVIEW

**Comment by joshgummersall** [comment]
The relevant dotnet PR is here: https://github.com/microsoft/botbuilder-dotnet/pull/4997

**Comment by joshgummersall** [comment]
Here is a sample issue from testing against some of my personal repos: https://github.com/joshgummersall/ts-plugin-mvp/issues/29

**Review by stevengum — APPROVED** [review]
This is a good start, in the future we might have to write a custom workflow to get to where we want (i.e. if some change occurs in the *.sln or *.csproj files in .NET we likely don't need those changes in JS or Python)

Also, it seems like it might be possible to get the PRs linked issue according to this [this doc page](https://docs.github.com/en/free-pro-team@latest/rest/reference/pulls).

(Reached via https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/webhook-events-and-payloads#pull_request)

**Comment by stevengum** [comment]
@EricDahlvang FYI

**Comment by joshgummersall** [comment]
Any repo that is "downstream" of `botbuilder-dotnet` can copy our template here and augment the [workflow file](https://github.com/microsoft/botbuilder-dotnet/pull/4997/files#diff-f699dfd7fe2e4cb3553ebda54997b8b492a7eaf1fc49f76fb3bb1a35d00b7e47R16) to trigger.

**Comment by joshgummersall** [comment]
@stevengum my goal was to use the simplest primitives possible in the bb-js repo and rely on upstream producers to determine when to trigger this action. Specifically, we could adjust the PR merge workflow in the `botbuilder-dotnet` to filter certain files via globs, or the like. This is basically an API endpoint for other things to use when they think we ought to have a new parity issue opened. I could possibly even adjust it to accept a title, description, and ref instead of being pull request specific to support issues as well.

File: .github/workflows/create-parity-issue.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+jobs:
+  createIssue:
+    name: create issue
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: joshgummersall/create-issue@main
```
Comment (joshgummersall) [inline .github/workflows/create-parity-issue.yml:28]: @gabog this action might be useful in writing a workflow to automatically create child issues from a parent botframework-sdk issue.

**Pull request merged**